package.json

```diff
@@ -30,7 +30,7 @@
     "react-router-dom": "^6.4.2",
     "uuid": "^9.0.0",
     "vite-plugin-package-version": "^1.0.2",
-    "warp-contracts": "1.2.35"
+    "warp-contracts": "^1.2.52"
```

Comment:
😬

Reply:
wow. I didnt think that was possible.

Reply:
pinned warp.

This was updated because a package dep in warp was updated.